ThirdPartyAdapters/vungle/vungle/src/main/java/com/google/ads/mediation/vungle/VungleMediationAdapter.java

```diff
@@ -335,6 +335,11 @@ public void run() {
         });
   }
 
+  @Override
+  public void onAdViewed(String id) {
+    // no op for now
```

Comment:
can we change the wording as "no-op , to be mapped to respective adapter events in future release"